styleguide/derek/templates/landing-page-noBanner/landingpage-single-webinar.ejs

```diff
@@ -0,0 +1,117 @@
+          <li>I'm a list item</li>
+          <li>I'm a list item</li>
+        </ul>
+      </p>
+      <p>
+        Beyond empowering companies to tap into the power of the cloud, Rackspace also strives for social 
+        empowerment. In that spirit, we’ll make a monetary donation of $15 per attendee to Girls Who Code – 
+        up to $5,000. In addition to a great day of learning and networking, your attendance at Rackspace::Solve 
+        will help give high school girls the skills and resources they need to pursue opportunities in computing 
+        fields.
+      </p>
+    </div>
+    <div class="event-form">
+      <h2>Register for the Webinar</h2>
+      <!-- NOTE: if a form is present, there should be NO CTA. If form IS present, CTA should be there -->
+      <div class="calloutForm">
+        <form>
+          <div class="form-group">
+            <label for="firstName" class="form-label">First Name</label>
+            <input type="name" class="form-control" id="firstName">
+          </div>
+          <div class="form-group">
+            <label for="lastName" class="form-label">Last Name</label>
+            <input type="name" class="form-control" id="lastName">
+          </div>
+          <div class="form-group">
+            <label for="email" class="form-label">Email address</label>
+            <input type="email" class="form-control" id="email">
+          </div>
+          <div class="form-group">
+            <label for="phone" class="form-label">Phone Number</label>
+            <input type="phone" class="form-control" id="phone">
+          </div>
+
+          <img src="/imgs/derek/captcha.jpg" style="max-width: none; width: 304px; height: 78px; margin: 10px 0;">
+
+          <div class="form-actions">
+            <button type="submit" class="button lead form-submit">Send</button>
+          </div>
+          <p class="small half-padding-top">Information collected in this form is subject to the <a href="#">Rackspace Privacy Statement.</a></p>
+        </form>
+      </div>
+      <div class="videoEmbed-wrapper">
+        <%- partial('../../pattern-components/video/video.ejs') %>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+<div class="event-moderators-section">
+  <div class="event-moderators-container">
+    <h3 class="event-moderators-header">Moderators</h3>
+    <% for(var i = 0; i < 2; i++) { %>
+      <div class="event-moderators-item">
+        <div class="event-moderators-picContainer">
+          <img class="event-moderators-pic" src="../imgs/derek/speaker-thumb.jpg" alt="">
+        </div>
+        <div class="event-moderators-info">
+          <h3 class="event-moderators-name">Derek Zoolander</h3>
+          <p>Supermodel, Los Angeles</p>
+          <p>Mugatu Inc.</p>
+          <p>
+            Street art dreamcatcher cronut typewriter tumblr selfies. Kogi single-origin coffee raw denim,
+            retro crucifix 90's tacos polaroid marfa wolf. Whatever disrupt normcore actually af. Roof party
```

Comment:
You mind dropping the apostrophe here so we can keep github from going crazy? That's a rather annoying bug on GitHub's end.